### PR TITLE
Operations

### PR DESCRIPTION
**Pull request by cyonii**
In this milestone I've implemente the logic for the calculations and operations.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operator) {
+  const operatorsMap = {
+    '÷': Big.prototype.div,
+    '*': Big.prototype.times,
+    '+': Big.prototype.plus,
+    '-': Big.prototype.minus,
+    '%': Big.prototype.mod,
+  };
```
Comment (trekab) [inline src/logic/operate.js]: The `%` operator should be turning the number into a percentage based number, e.g. 95 % == .95 ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png))

**Review by trekab — CHANGES_REQUESTED** [review]
### Changes Required! :recycle: 
Hi @cyonii! Nice work with the logic modules so far. There are a few requirements I'd like you to consider before finishing up with this milestone though. i.e;
- [ ] Import `calculate.js` in your `App` component.
- [ ] Your `%` operator seems to be different from the expected logic. Check the comment in the code below for more details.

_Please make the necessary changes and submit another code review request._
__[Treasure Kabareebe](https://trekab.github.io)__

File: src/logic/operate.js (hunk 1 above)
Comment (cyonii) [reply src/logic/operate.js]: Oh, Thank you. Fixed.

**Review by trekab — APPROVED** [review]
### Approved! :tada: 
Nicely done @cyonii! :+1: You may proceed to your next milestone.

_Happy coding!_
__[Treasure Kabareebe](https://trekab.github.io)__

**Pull request merged**